docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Ingress Controller
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc kubernetes Ingress reverse-proxy
+---
+
+# Kubernetes Ingress Controller
+
+Use Pomerium as a first-class secure-by-default Ingress Controller. Dynamically provision routes from Ingress resources and set policy based on annotations. By defining routes as Ingress resources you can independently create and remove them from Pomerium's configuration. This enables workflows more native to Kubernetes environments such as actions based on pull requests.
+
+## Prerequisites
+
+- A certificate management solution. If you do not already have one in place, this article covers using Cert Manager.
+- A Redis backend with high-persistence is highly recommended.
+
+### System Requirements
+
+- Kubernetes v1.19.0+
+- Pomerium [Helm Chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) v25.0.0+
+
+### Limitations
+
+::: warning
+
+Only one Ingress Controller instance/replica is supported per Pomerium cluster.
+
+:::
+
+## Installation
+
+### Helm
+
+Our instructions for [Installing Pomerium Using Helm](/docs/k8s/helm.md) includes the Ingress Controller as part of the documented configuration.
+
+
+```yaml
+ingressController:
+  enabled: true
+```
+
+### Docker Image
+
+You may deploy your own manifestations by using the `pomerium/ingress-controller` docker image.
```

Comment:
What is a manifestation?

Reply:
It doesn't appear to be a term used in other talk about Docker or Kubernetes. Unless @travisgroth has any opinion on this, I'll change it to:

```suggestion
You may deploy the Ingress controller as part of a custom [manifest](https://docs.docker.com/engine/reference/commandline/manifest/) by using the `pomerium/ingress-controller` docker image.
```

Reply:
Docs link should be to kubernetes - not docker - or just not a link.    

```suggestion
You may deploy the Ingress controller from your own manifests by using the `pomerium/ingress-controller` docker image.
```